relay/kusama/src/ah_migration/phase1.rs

```diff
@@ -0,0 +1,177 @@
+/// 2. After the migration: (End, ∞)
+///
+/// Visually:
+///
+/// ```text
+///         |-----1-----|
+///                      |---2---->
+/// --------|-----------|--------->
+///       Start        End
+/// ```
+///
+/// This call returns a 2-tuple to indicate whether a call is enabled during these periods.
+pub fn call_allowed_status(call: &<Runtime as frame_system::Config>::RuntimeCall) -> (bool, bool) {
+	use RuntimeCall::*;
+	const ON: bool = true;
+	const OFF: bool = false;
+
+	match call {
+		System(..) => (ON, ON), // Remarks, root calls and `set_code` if we need for emergency.
+		Scheduler(..) => (OFF, OFF), // Only for governance, hence disabled.
+		Preimage(..) => (OFF, OFF), // Only for governance, hence disabled.
+		Babe(..) => (ON, ON),   // For equivocation proof submissions; security relevant
+		Timestamp(..) => (ON, ON), // only `set` inherit
+		Indices(..) => (OFF, OFF), // Not needed anymore and migrated to AH.
+		Balances(..) => (OFF, ON), // Disabled during migration to avoid confusing externals.
+		Staking(..) => (OFF, OFF),
+		StakingAhClient(..) => (ON, ON), // Only permissioned calls and needed for the migration.
+		Session(..) => (ON, ON),         // Does not affect any migrating pallet.
+		Grandpa(..) => (ON, ON),         // For equivocation proof submissions; security relevant
+		Treasury(..) => (OFF, OFF),
+		ConvictionVoting(..) => (OFF, OFF),
+		Referenda(..) => (OFF, OFF),
+		Whitelist(..) => (OFF, OFF),
+		Claims(..) => (OFF, OFF),
+		Vesting(..) => (OFF, OFF),
+		Utility(..) => (ON, ON),   // batching etc
+		Proxy(..) => (OFF, ON),    // On after the migration to keep proxy accounts accessible.
+		Multisig(..) => (OFF, ON), // On after the migration to keep multisig accounts accessible.
+		Bounties(..) => (OFF, OFF),
+		ChildBounties(..) => (OFF, OFF),
+		ElectionProviderMultiPhase(..) => (OFF, OFF),
+		VoterList(..) => (OFF, OFF),
+		NominationPools(..) => (OFF, OFF),
+		FastUnstake(..) => (OFF, OFF),
+		Configuration(..) => (ON, ON), /* TODO allow this to be called by fellow origin during the migration https://github.com/polkadot-fellows/runtimes/pull/559#discussion_r1928794490 */
+		ParasShared(parachains_shared::Call::__Ignore { .. }) => (ON, ON), // Has no calls
+		ParaInclusion(parachains_inclusion::Call::__Ignore { .. }) => (ON, ON), // Has no calls
+		ParaInherent(..) => (ON, ON),  // only inherents
+		Paras(..) => (ON, ON),         /* Only root and one security relevant call: */
+		// `include_pvf_check_statement`
+		Initializer(..) => (ON, ON), // Only root calls. Fine to keep.
+		Hrmp(..) => (ON, ON),        /* open close hrmp channels by parachains or root force. */
+		// no concerns.
+		ParasDisputes(..) => (ON, ON), // Only a single root call. Fine to keep.
+		ParasSlashing(..) => (ON, ON), /* Security critical. If disabled there will be no */
+		// slashes or offences generated for malicious
+		// validators.
+		OnDemandAssignmentProvider(..) => (OFF, ON),
```

Comment:
why off during the migration? isn't this the call that is dispatched by an XCM via the coretime chain?

Reply:
Donal did review those AFAIK, but i cant answer your Q...